compiler/luci-micro/luci-interpreter/src/loader/GraphLoader.cpp

```diff
@@ -109,18 +112,22 @@ void GraphLoader::initInputTensors() const
       shape.dim(i) = tensor_shape.at(i);
     }
 
-    AffineQuantization quantization;
+    AffineQuantization *quantization = nullptr;
     if (dtype == DataType::U8 or dtype == DataType::S8 or dtype == DataType::S16)
     {
+      auto unique_ptr_quantization = std::make_unique<AffineQuantization>();
       const auto quant_params = tensor->quantization();
       assert(quant_params->zero_point()->size() == quant_params->scale()->size());
-      quantization.scale.assign(quant_params->scale()->cbegin(), quant_params->scale()->cend());
-      quantization.zero_point.assign(quant_params->zero_point()->cbegin(),
-                                     quant_params->zero_point()->cend());
-      quantization.quantized_dimension = quant_params->quantized_dimension();
+      unique_ptr_quantization->scale.assign(quant_params->scale()->cbegin(),
+                                            quant_params->scale()->cend());
+      unique_ptr_quantization->zero_point.assign(quant_params->zero_point()->cbegin(),
+                                                 quant_params->zero_point()->cend());
+      unique_ptr_quantization->quantized_dimension = quant_params->quantized_dimension();
+
+      quantization = _runtime_graph->addAffineQuantization(std::move(unique_ptr_quantization));
```

Comment:
optinal: move this code with same code from `GraphLoader::loadTensors()` in separate function